recipes/util-linux/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "util-linux" %}
+{% set version = "2.30" %}
+{% set sha256 = "e84cc32566abf28b84efb5ee326e1f3597c3d19b9916740f50c75bd57e498ce0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    # Skipping osx as this is really a linux only library. (util-linux in the name is a give-away..)
+    skip: True  # [win or osx]
+    
+requirements:
+    build:
+        - toolchain
+        - automake
+        - autoconf
+        - bison
+        - pkg-config
+        - libtool
+        - gettext
+    run:
+        - libgcc
```

Comment:
This `run` portion can be dropped. We use the CentOS 6 system's `libgcc` currently.